docs/rst/tutorials/core/qos/xml_profiles/xml_profiles.rst

```diff
@@ -0,0 +1,400 @@
+.. include:: ../../../../exports/alias.include
+
+.. _tutorials_xml_profiles_intro:
+
+Configuring Fast-DDS QoS via XML profiles
+=========================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+XML configuration
+-----------------
+
+Fast DDS over Vulcanexus offers the possibility of fully configuring QoS policy through XML profile definition.
+This can be done in several ways depending on whether the application used is a native Fast DDS application or a ROS 2 application
```

Comment:
```suggestion
This can be done in several ways depending on whether the application used is a native Fast DDS application or a Vulcanexus application
```

Reply:
This is not addressed